### PR TITLE
Remove old AT's and just run the new ones

### PR DESCRIPTION
**Pull request by markxnelson**
This PR removes the old merge tests job, and now runs the new AT's.
It also introduces some parallelization and a few small cleanups to make the job a little faster.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -335,182 +336,168 @@ pipeline {
...
+                            kubectl -n verrazzano-install rollout status deployment/verrazzano-platform-operator
+
+                            echo "Installing Verrazzano on Kind"
+                            kubectl apply -f ${INSTALL_CONFIG_FILE_KIND}
+
+                            # wait for Verrazzano install to complete
+                            ./tests/e2e/config/scripts/wait-for-verrazzano-install.sh
+
+                            # Hack
+                            # OCIR images don't work with KIND.
```
Comment (gilbode) [inline Jenkinsfile:404]: The comment says OCIR, this is OCR

File: Jenkinsfile [hunk 2]
```diff
@@ -335,182 +336,168 @@ pipeline {
...
-                                    dumpK8sCluster('new-acceptance-tests-cluster-dump.tar.gz')
-                                }
-                            }
+                    }
+                }
+            }
+
+            post {
+                failure {
+                    dumpK8sCluster('new-acceptance-tests-cluster-dump.tar.gz')
```
Comment (gilbode) [inline Jenkinsfile:495]: Do you now have to download & unpack the *.tgz with this or will you still be able to see individual files from Jenkins?

File: Jenkinsfile [hunk 3]
```diff
@@ -335,182 +336,168 @@ pipeline {
...
-                        build job: "verrazzano-merge-tests/${env.BRANCH_NAME.replace("/", "%2F")}",
-                                parameters: [string(name: 'VERRAZZANO_BRANCH', value: env.BRANCH_NAME),
-                                             string(name: 'ACCEPTANCE_TESTS_BRANCH', value: params.ACCEPTANCE_TESTS_BRANCH),
-                                             string(name: 'VERRAZZANO_OPERATOR_IMAGE', value: FULL_IMAGE_NAME),
-                                             string(name: 'TEST_ENV', value: 'kind'),
-                                             string(name: 'INSTALL_PROFILE', value: 'dev')],
-                                wait: true,
-                                propagate: true
+                        sh """
+                            echo "Create Kind clsuter"
```
Comment (agv426) [inline Jenkinsfile:355]: nit: typo

File: Jenkinsfile (hunk 2 above)
Comment (gilbode) [reply Jenkinsfile:495]: It seems you do - can't we just archive all the individual files?

Comment (markxnelson) [reply Jenkinsfile:495]: it shows up in the jenkins artifacts, i just click on it to download and open, it works well

File: Jenkinsfile (hunk 3 above)
Comment (markxnelson) [reply Jenkinsfile]: fixed

File: Jenkinsfile (hunk 1 above)
Comment (markxnelson) [reply Jenkinsfile]: fixed

File: Jenkinsfile (hunk 2 above)
Comment (agv426) [reply Jenkinsfile:495]: these were added earlier (in our current jobs now). These are a single in jenkins currently (the "works"). We could expand them out. Wasn't sure we wanted to do that up front though and if disk space is a concern.

**Review by agv426 — APPROVED** [review]

**Review by gilbode — APPROVED** [review]

File: Jenkinsfile [hunk 4]
```diff
@@ -335,182 +336,168 @@ pipeline {
...
+                        branch 'master';
+                        branch 'develop';
+                        expression {SKIP_ACCEPTANCE_TESTS == false};
                     }
+                }
+            }
+
+            stages {
+
+                stage('Prepare AT environment') {
```
Comment (pfmackin) [inline Jenkinsfile:352]: when i looked at  a failed pipeline it wasn't obvious that this stage installs verrazzano (which failed)

**Pull request merged**